src/app/compare/Compare.tsx

```diff
@@ -304,12 +304,27 @@ export default function Compare({ session }: { session: CompareSession }) {
         />
       )}
       {exampleItem && <CompareSuggestions exampleItem={exampleItem} onQueryChanged={updateQuery} />}
-      {organizerLink && (
-        <Link className={styles.organizerLink} to={organizerLink}>
-          <AppIcon icon={faList} />
-          <span>{t('Organizer.OpenIn')}</span>
-        </Link>
-      )}
+      <div className={styles.rightOptions}>
+        <button
+          type="button"
+          className="dim-button"
+          onClick={() => {
+            for (const item of compareItems) {
+              if (item !== initialItem) {
+                remove(item);
+              }
+            }
+          }}
```

Comment:
I generally prefer to pull click handlers out into named functions instead of having them inline.